tests/series_test.py

```diff
@@ -7,21 +7,52 @@
 class SeriesTest(ArkoudaTest):
 
     def test_series_creation(self):
-        ar_tuple = (ak.arange(3), ak.array(['A', 'B', 'C']))
-        s = ak.Series(ar_tuple=ar_tuple)
+        # Use positional arguments
+        ar_tuple = ak.array(ak.arange(3)), ak.array(ak.arange(3))
```

Comment:
`ak.arange` returns a pdarray, so you shouldn't need to wrap it in an `ak.array`

I think this line can be changed to 
```python
ar_tuple = ak.arange(3), ak.arange(3)
```